theme/highlight.css

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-marker-yellow: #fdfd77;
```

Comment:
I'd prefix it with `--ck-highlight-(marker|pen)-` to stick to the convention we use.